tests/test_main.py

```diff
@@ -10,7 +10,9 @@
 def path_to_main():
     base_path = os.path.join("app", "main.py")
     return (
-        base_path if os.path.exists(base_path) else os.path.join(os.pardir, base_path)
+        base_path
+        if os.path.exists(base_path)
+        else os.path.join(os.pardir, base_path)
```

Comment:
why?

Reply:
accidentally applied linter for all files🥲